hal/src/tron/network/network.cpp

```diff
@@ -119,11 +119,7 @@ int if_init_platform(void*) {
     }
 
     if (HAL_Feature_Get(FEATURE_ETHERNET_DETECTION)) {
-#if PLATFORM_ID == PLATFORM_ARGON || PLATFORM_ID == PLATFORM_P2
         en2 = new WizNetif(HAL_SPI_INTERFACE1, D5, D3, D4, mac);
```

Comment:
See tracker m network.cpp for comments around mac address for ethernet interface.